js/stores/wizard_store.js

```diff
@@ -8,11 +8,16 @@
 import { create } from 'zustand';
 import { shallow } from 'zustand/shallow';
 import { fetchWizardInitData, fetchWizardPredictions } from '../util/wizard_api';
-import { convertSomeLinksToCards, normalizeScore, urlParams } from '../util/wizard_helpers';
+import {
+  convertSomeLinksToCards, normalizeScore, scanForTriggers, urlParams,
+} from '../util/wizard_helpers';
 import allTopics from '../models/wizard_topics';
 import extraMessages from '../models/wizard_extra_messages';
 import { defaultSummary, stateLocalSummary, stateOrLocalFlow } from '../models/wizard_summaries';
 
+/** @type {WizardTriggerPhrase[]} */
+let triggerPrases = [];
```

Comment:
I might consider changing these instances to `triggerPhrases`, but it's inconsequential